src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react'
+import { MIGRATION_ADDRESS } from 'src/routes/routes'
+import { saveToStorage, loadFromStorage } from 'src/utils/storage'
+
+const MIGRATION_PATH_BASE_URL = 'http://localhost:3001/#'
```

Comment:
Germán added a new config value, APP_DOMAIN, in his PR. Once it's merged, we can use it here for prod. And an `*` for non-prod envs.